README.md

```diff
@@ -49,6 +49,23 @@ const printNodeVersions = async function (options) {
   //   { major: 4, latest: '4.9.1', lts: 'argon' },
   //   { major: 0, latest: '0.12.18' }
   // ]
+
+  console.log(nodeNpmVersion)
```

Comment:
Instead of returning a different export `nodeNpmVersions`, what are your thoughts on making the existing `versions` export an array of objects instead of of an array of strings instead? This would be more extensible to additional fields in the future, such as `v8` or `modules`.

```js
const { versions, majors } = await allNodeVersions(options)
console.log(versions)
// [
//   { node: '16.3.0', npm: '8.11.0' },
//   ...
// ]
```

Reply:
Well, that would work for me, I just was conscious about introducing a breaking change... Because previously it was an array of strings, we'll have to bump the major version, let me know if that's fine by you?

Reply:
Yep, it would be a breaking change indeed and a major release. :+1: